frontend/apps/marketing/src/components/carousels/videoCarousel/VideoCarousel.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+
+import '@code-dot-org/component-library/carousel/index.css';
+import DSCOCarousel, {
+  CarouselProps,
+} from '@code-dot-org/component-library/carousel';
+import React, {ReactNode} from 'react';
+
+import Video from '@code-dot-org/component-library/video';
+
+// Import Swiper styles
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
```

Comment:
non blocking, just for context: I see that we import these styles in Carousel in component-library itself. I wonder if we need to also import this here, if yes - why?

Reply:
Everything looks fine if we remove these so I think this was a leftover from tinkering, I'll remove them!